app/templates/partials/questions/repeatinganswer.html

```diff
@@ -20,7 +20,7 @@ <h3 class="neptune">{{ _('Person ') }}<span class="js-household-loopindex">{{ lo
           {%- if loop.index == 1 -%}
              <span class="js-household-action">{{ _('You') }}</span>
           {%- else -%}
-            <button class="btn btn--link pluto js-btn-remove" name="action[remove_answer]" value="{{ loop.index0 }}" type="button">{{ _('Remove') }}
+            <button class="btn btn--link pluto js-btn-remove" name="action[remove_answer]" value="{{ loop.index0 }}" type="submit">{{ _('Remove') }}
```

Comment:
For non-JS enabled browsers, this is what was causing people (specifically Person 2) to be removed when users press ENTER key. So just to be aware that the defect will be re-introduced for non-JS browsers with this change.